OneBusAway/OBAApplicationDelegate.m

```diff
@@ -219,6 +219,21 @@ + (OBADeepLinkRouter*)setupDeepLinkRouterWithModelDAO:(OBAModelDAO*)modelDAO app
 }
 
 - (BOOL)application:(UIApplication *)application continueUserActivity:(NSUserActivity *)userActivity restorationHandler:(void (^)(NSArray *))restorationHandler {
+    NSInteger regionID = [userActivity.userInfo[[OBAHandoff regionIDKey]] integerValue];
+
+    if (userActivity.userInfo != nil) {
```

Comment:
I'm leery of this having negative effects on the deep linking stuff below. Please restructure this along the following lines:

```
// Handoff
if (userActivity.userInfo) {
  // ... do stuff
  return YES;
}

// do other stuff.
```